README.md

```diff
@@ -1,7 +1,7 @@
 # XLT
-XLT is an extensive load and performance test tool developed and maintained by Xceptance. If you need more information, here is the current website https://www.xceptance.com/xlt/ and the current documentation portal https://lab.xceptance.de/releases/xlt/latest/index.html. There is also a forum available to discuss load testing and test automation with XLT: https://ask.xceptance.de/
+XLT is an extensive load and performance test tool developed and maintained by Xceptance. If you need more information, here is the current website https://www.xceptance.com/xlt/ and the current documentation https://xltdoc.xceptance.com/. There is also a forum available to discuss load testing and test automation with XLT: https://ask.xceptance.de/
 
-There is a major update to the documentation in progress. You can contribute on GitHub https://github.com/Xceptance/xlt-documentation and find the deployed documentation at https://xltdoc.xceptance.com/
+There documentation has been rebuilt in 2020 and is still being updated regularly. You can contribute on GitHub https://github.com/Xceptance/xlt-documentation. If you need any information from the legacy documentation portal you can still find it at https://lab.xceptance.de/releases/xlt/latest/index.html
```

Comment:
The documentation was full rewritten in 2020 and it is continuously updated. We appreciated your feedback.You can also directly contribute on GitHub https://github.com/Xceptance/xlt-documentation. If you need any information from the legacy documentation, you can still find it at https://lab.xceptance.de/releases/xlt/latest/index.html

Reply:
So be it (minus the typos)